Userland/Libraries/LibVideo/VP9/Symbols.h

```diff
@@ -20,7 +20,8 @@ namespace Video::VP9 {
 #define MIN_TILE_WIDTH_B64 4
 #define MAX_TILE_WIDTH_B64 64
 #define MAX_MV_REF_CANDIDATES 2
-#define NUM_REF_FRAMES 8
+#define LOG2_OF_NUM_REF_FRAMES 3
```

Comment:
Wew. It would be nice to group these into enums and static constexpr variables at some point. And to reduce their visibility to the headers that actually need them instead of one big "MyDefines.h"

Reply:
Agreed, I'll add a fixme to that commit so that I or someone else can hopefully get to that.